README.md

```diff
@@ -1,2 +1,165 @@
-# kss-caxy-zaba-template
-Caxy's Zaba kss-node builder template.
+# Caxy's Zaba Theme KSS Builder
+
+TODO: How to Generate Styleguide and Viewing the Styleguide sections
+likely need revision.
```

Comment:
Yeah! This is meant to be just a kss-node "builder" (i think that's what they're called) - so it would be good to add documentation of how to use this "builder" with kss-node. And I think all that might be is how to use it when using a json config file, setting the "builder" key to point to the path of this package ... and also how to use it when using kss-node on the command line (if there's a builder cli option you can pass).

I think actually generating a styleguide from this package will either be outside the scope of this repo, OR we might be able to set up dev-only tools to do so in this package.

Either way I think adding Github issues for this would be good